### PR TITLE
net-misc/google-cloud-sdk: add gcloud and gsutil

### PR DESCRIPTION
**Pull request by crawford**

### CODE REVIEW

**Comment by marineam** [comment]
Is this for use in the SDK? How does packaging this mesh with the fact that gcloud is it itself a package manager?

File: net-misc/google-cloud-sdk/google-cloud-sdk-108.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+KEYWORDS="amd64"
+IUSE=""
+
+S="${WORKDIR}/${PN}"
+
+DEPEND="${PYTHON_DEPS}"
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	epatch "${FILESDIR}/0001-Hardcode-the-root-directory.patch"
```
Comment (marineam) [inline net-misc/google-cloud-sdk/google-cloud-sdk-108.0.0.ebuild]: git add

**Comment by crawford** [comment]
@marineam yeah, for use in the SDK. The package management functionality just errors out.

```
$ gcloud components list
ERROR: (gcloud.components.list) The components management action could not be performed because the installation root of the Cloud SDK could not be located. If you previously used the Cloud SDK installer, you could re-install the the SDK and retry again.
```

**Comment by marineam** [comment]
Ok, as long as the magic doesn't do anything ;-)

LGTM

Though this isn't providing gsutil in the path but that can be a follow up, replacing the old gsutil ebuild.

**Pull request merged**